src/aap_eda/core/tasking/__init__.py

```diff
@@ -150,6 +150,9 @@ class ActivationWorker(_Worker):
     Uses JSONSerializer as a default one.
     """
 
+    # Assumption: Each process has only one ActivationWorker
```

Comment:
Is this assumption sustainable beyond the short term? I feel it might be an approach that we will have to remade sooner or later. I see potential problems with it: it might complicate tests if we want to instantiate more than a worker. It might not be compatible with [pooling](https://python-rq.org/docs/workers/#worker-pool), it will not support multi threading or asynchronous implementations. 

The Job already have a property named "worker_name".